snakeGame.js

```diff
@@ -1,101 +1 @@
-//new tab--canvas from javascript console
-var body = document.querySelector("body");
-var c = document.createElement("canvas");
-body.appendChild(c);
-let boardSize = 20;
-c.id = "c";
-c.width = boardSize * 10;
-c.height = boardSize * 10;
-c.style.border = "1px solid black";
-var ctx = c.getContext("2d");
 
-let state = {
-    snakeLength: 2,
-    appleLoc: vecInBoard(),
-    snakeLoc: [[boardSize/2, boardSize/2]],
-    snakeMove: "right"
-}
-
-window.addEventListener("keydown", recordInput);
-
-function recordInput(e) {
-    switch (e) {
-        case 37:
-            state.snakeMove = "left";
-        case 38:
-            state.snakeMove = "up";
-        case 39:
-            state.snakeMove = "right";
-        case 40:
-            state.snakeMove ="down";
-    }
-}
-
-tick();
```

Comment:
is there a loop or timer for running ticks?

Reply:
yes there is a loop for running ticks, but i didn't get there yet! :)